vendor/github.com/relationalai/rai-sdk-go/rai/models.go

```diff
@@ -38,9 +38,9 @@ type Database struct {
 }
 
 type EDB struct {
-	Name   string   `json:"name"`
-	Keys   []string `json:"keys"`
-	Values []string `json:"values"`
+	Name   string        `json:"name"`
+	Keys   []interface{} `json:"keys"`
```

Comment:
why interface?

Reply:
Not sure, didn't make this change myself. This was a vendoring update I pulled for 0.5.0-alpha

Reply:
oh right